### DISCUSSION TITLE
ESP32 audio effects

### DISCUSSION THREAD

**Discussion by bdphonehome** [pschatzmann/arduino-audio-tools#381] [Q&A] [answered]
Had anyone created a high quality reverb process using the ESP32?

I know that dedicated DSP devices and even the Teensy can do good reverb but I am looking to incorporate "guitar effects" and other tasks using the dual core ESP32 devices.

**Comment by pschatzmann** [comment, marked as answer]
I don't consider this to be an issue, so I convert it to a discussion!

As you can see in the [documentation](https://pschatzmann.github.io/arduino-audio-tools/html/classaudio__tools_1_1_audio_effect.html), you can use quite some reverb alternatives from STK. You can also consider to use the reverbs from Faust.

I let you judge, If these match your expectations!

It would be interesting to hear from somebody who have actually used any of this...
